packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uma-sdk/core",
-  "version": "0.8.2",
+  "version": "1.0.0",
```

Comment:
mmm @coreymartin is the right move for me to revert this and then actually just bump the version via changesets?

Reply:
Done.